src/cli/commands/bitswap/unwant.js

```
@@ -0,0 +1,24 @@
+'use strict'
+
+const Command = require('ronin').Command
+const utils = require('../../utils')
+
+module.exports = Command.extend({
+  desc: 'Remove a given block from your wantlist.',
+
+  options: {
+    key: {
+      required: true
+    }
+  },
+
+  run: (key) => {
+    utils.getIPFS((err, ipfs) => {
+      if (err) {
+        throw err
+      }
+
+      // TODO: implement
```

Let's throw a error here saying that it is not implemented yet